### ISSUE TITLE
Escaped new lines (\n) don't work after OTA update

### ISSUE THREAD

**Issue by bjoernQ** [phrase/phraseapp-android#8]
Thanks for fixing #7 - I unfortunately encountered a similar issue now: 

Escaped special characters like `\n` are not properly handled after an OTA update and show as `\n` instead of causing a line break

**Comment by theSoenke** [comment]
Hi, we are currently investigating this and will keep you updated

**Comment by theSoenke** [comment]
I was able to reproduce it and the fix is currently being completed. I will let you know as soon as a new release is available

**Comment by theSoenke** [comment]
We now released version 1.1.5 of the SDK. This should resolve the newline issue. Let me know if there is anything else should come up

**Comment by bjoernQ** [comment]
tried 1.1.5 and unfortunately it doesn't seem to fix my issue

after the OTA update a string containing `Hello\nWorld` becomes `Hello\\World` - so the backslash gets escaped while it shouldn't

**Comment by theSoenke** [comment]
Thanks for testing it. This was a slight misunderstanding and we solved another related issue. With version 1.1.5 non-escaped newlines will now be displayed correctly. Translations that explicitly contain `\n` will be escaped by default which is different behavior from normal strings files. This is the original root cause for this error. We will have a closer look at this and I will let you know when this has been fixed
